ladybug/header.py

```diff
@@ -145,7 +163,7 @@ def __iter__(self):
 
     def to_json(self):
         """Return a header as a dictionary."""
-        return {'data_type': self.data_type.to_json(),
+        return {'data_type': self.data_type.name,
```

Comment:
I can add a to_json() method but it will essentially return the name of the DataType since all that is needed to recreate any data type is the name and an acceptable unit.

Reply:
I put back the from_json and to_json methods.  It was a bit of a challenge to do this without having methods that recall data type instances by name but, thanks to some dictionaries that are in the _enumeration object, this was made possible.